apps/passport-client/public/global-zupass.css

```diff
@@ -156,3 +156,59 @@ img {
   user-select: none;
   user-drag: none;
 }
+
```

Comment:
add a comment explaining why this is handled in this css file rather than inside of the app (because it needs to work without the js bundle having loaded.